backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -116,9 +116,6 @@ public IActionResult GetMyUsers(EModel.UserFilter filter)
         public IActionResult GetUser(Guid id)
         {
             var entity = _pimsAdminService.User.Get(id);
-
-            if (entity == null) return NoContent();
```

Comment:
Not needed as the Get will throw a KeyNotFoundException which is handled by our middlelayer which will then return a BadRequest.

Reply:
Perhaps consider leaving a comment here so we don't add this logic back?